### ISSUE TITLE
Parsing error: Unexpected token <

### ISSUE THREAD

**Issue by PLQin** [vuejs/eslint-plugin-vue#746]
**Tell us about your environment**

* **ESLint Version:** v5.11.1
* **Node Version:** v10.14.2
* **npm Version:** 6.4.1

**What parser (default, Babel-ESLint, etc.) are you using?**

- .eslintrc.js

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
module.exports = {
    root: true,
    parserOptions: {
        "parser": "babel-eslint", 
        "sourceType": "module",
        "ecmaVersion": 6
    },
    env: {
        browser: true,
        node: true,
        es6: true,
    },
    extends: ["eslint:recommended"],

    // add your custom rules here
    //it is base on https://github.com/vuejs/eslint-config-vue
    rules: {
        "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
        "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",
        "no-undef": "off",
        "no-unused-vars": "off",
        "no-redeclare": "off",
        "no-empty": "off",
        "indent": ["warn", 4],
        'vue/max-attributes-per-line': 'off', 
        'vue/order-in-components': 'off',
        'vue/html-end-tags': 'off',
    }
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

vue code
```
<template>
  <div id="app">
    <router-view></router-view>
  </div>
</template>

<script>
export default {
  name: 'App' 
};
</script>
```

run ESLint : 

```bash
eslint --fix "./src/**/*.{js,vue}"
```

**What did you expect to happen?**

good running 

**What actually happened? Please include the actual, raw output from ESLint.**

```
D:\_Me\-COMPANY-JiTeng-\src\App.vue : 
 1:1  error  Parsing error: Unexpected token <
```

**Are you willing to submit a pull request to fix this bug?**

![image](https://user-images.githubusercontent.com/24371471/50627561-f0115e00-0f6e-11e9-8d67-efed11f368ed.png)

**Comment by armano2** [comment]
you are missing parser

**Comment by plqin-xiaohao** [comment]
> you are missing parser

`vue-loader` ? 

I'm using;

Also, not using ` eslint `, I compile properly ; Project normal use, normal packaging

**Comment by michalsnik** [comment]
Hi @PLQin! Please take a look at the installation instruction https://eslint.vuejs.org/user-guide/#installation
You're not extending configuration that this plugin provides.

**Comment by plqin-xiaohao** [comment]
> Hi @PLQin! Please take a look at the installation instruction https://eslint.vuejs.org/user-guide/#installation
> You're not extending configuration that this plugin provides.

`eslint-plugin-vue` is mounted : 
` "eslint-plugin-vue": "4.7.1", `

But I didn't use ` "plugin:vue/recommended" `, I chose the ` "eslint:recommended" ` ;

My guess is that the use of ` "plugin:vue/recommended" ` should can solve my problem, I'll go to have a try .

But I don't want to use most of  ` "plugin:vue/recommended" ` rule , how should I do, could you please ` plugin:vue/recommended` rules of initialization, defined by me;

Thank you very much for your response

**Comment by ota-meshi** [comment]
You can use the following settings.

`"extends": ["plugin:vue/base"]`

https://eslint.vuejs.org/user-guide/#faq

**Comment by michalsnik** [comment]
There are few categories of rules available, if you don't like recommended, you can use `base`, `essential` or `strongly-recommended` instead. Please look at available configs and rules in each of those here: https://eslint.vuejs.org/rules/

**Comment by plqin-xiaohao** [comment]
> You can use the following settings.
> 
> `"extends": ["plugin:vue/base"]`
> 
> https://eslint.vuejs.org/user-guide/#faq



> There are few categories of rules available, if you don't like recommended, you can use `base`, `essential` or `strongly-recommended` instead. Please look at available configs and rules in each of those here: https://eslint.vuejs.org/rules/


thanks, thank you ; 

I'll try it right away. If it works, I'll give you feedback right away

**Comment by PLQin** [comment]
Nice, have good, through the set to ` "extends" : [" plugin: vue/base "] ` solved my problem ;

thank you !~ 👍

**Comment by touxing** [comment]
> You can use the following settings.
> 
> `"extends": ["plugin:vue/base"]`
> 
> https://eslint.vuejs.org/user-guide/#faq

Thanks ，It's solved my problem.

**Comment by touxing** [comment]
> You can use the following settings.
> 
> `"extends": ["plugin:vue/base"]`
> 
> https://eslint.vuejs.org/user-guide/#faq

Thanks, It's solved my problem too.

**Comment by disocom** [comment]
"extends": ["plugin:vue/base"

Thanks, It's solved my problem too.

**Comment by nirnejak** [comment]
Thanks A lot

**Comment by iamanas20** [comment]
`"extends": ["plugin:vue/base"]`

This solved it for me! Man what a community (●'◡'●)! Thank you!

**Comment by kabaluyot** [comment]
thankkk you <3

**Comment by yousuff-tekhne** [comment]
> You can use the following settings.
> 
> `"extends": ["plugin:vue/base"]`
> 
> https://eslint.vuejs.org/user-guide/#faq

Thank you <3000

**Comment by AndersonJo** [comment]
> You can use the following settings.
> 
> `"extends": ["plugin:vue/base"]`
> 
> https://eslint.vuejs.org/user-guide/#faq

Thanks! it works like charm!

**Comment by shaobeichen** [comment]
```
parser: 'vue-eslint-parser', // Adding this line works perfectly
extends: [
        'plugin:vue/base',
        'eslint:recommended',
        'plugin:vue/vue3-recommended', 
        'plugin:vue/essential',
        'plugin:@typescript-eslint/recommended',     // caused by this line
        'plugin:prettier/recommended',
        'eslint-config-prettier'
    ],
```

**Comment by citrusjunoss** [comment]
it's not work

`module.export = {
  env: {
    browser: true,
    node: true,
    es6: true,
  },
  parser: 'vue-eslint-parser',
  parserOptions: {
    parser: '@typescript-eslint/parser',
    ecmaVersion: 2020,
    sourceType: 'module',
    project: './tsconfig.*?.json',
    createDefaultProgram: false,
    extraFileExtensions: ['.vue'],
  },
  extends: [
    'eslint:recommended',
    'plugin:vue/vue3-recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:prettier/recommended',
    "plugin:jsdoc/recommended"
  ],
  rules: {
    'no-unused-vars': 'off',
    'no-case-declarations': 'off',
    'no-use-before-define': 'off',
    'space-before-function-paren': 'off',
    'import/first': 'error',
    'import/newline-after-import': 'error',
    'import/no-duplicates': 'error',
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        argsIgnorePattern: '^_',
        varsIgnorePattern: '^_',
      },
    ],
    '@typescript-eslint/ban-ts-ignore': 'off',
    '@typescript-eslint/ban-ts-comment': 'off',
    '@typescript-eslint/ban-types': 'off',
    '@typescript-eslint/explicit-function-return-type': 'off',
    '@typescript-eslint/no-explicit-any': 'off',
    '@typescript-eslint/no-var-requires': 'off',
    '@typescript-eslint/no-empty-function': 'off',
    '@typescript-eslint/no-use-before-define': 'off',
    '@typescript-eslint/no-non-null-assertion': 'off',
    '@typescript-eslint/explicit-module-boundary-types': 'off',
    'vue/script-setup-uses-vars': 'error',
    'vue/no-reserved-component-names': 'off',
    'vue/custom-event-name-casing': 'off',
    'vue/attributes-order': 'off',
    'vue/one-component-per-file': 'off',
    'vue/html-closing-bracket-newline': 'off',
    'vue/max-attributes-per-line': 'off',
    'vue/multiline-html-element-content-newline': 'off',
    'vue/singleline-html-element-content-newline': 'off',
    'vue/attribute-hyphenation': 'off',
    'vue/require-default-prop': 'off',
    'vue/require-explicit-emits': 'off',
    'vue/html-self-closing': [
      'error',
      {
        html: {
          void: 'always',
          normal: 'never',
          component: 'always',
        },
        svg: 'always',
        math: 'always',
      },
    ],
    'vue/multi-word-component-names': 'off',
    "max-lines": [
      "error",
      {
        "max": 500, 
        "skipBlankLines": true,
        "skipComments": true
        }
    ],
    "max-depth": ["error", 4],
    "max-lines-per-function": [
      "error", 
      {
        "max": 20, 
        "skipBlankLines": true, 
        "skipComments": true
      }
    ],
    "max-nested-callbacks": ["error", 5]
  },
  globals: { defineOptions: 'readonly' },
};`


what should I do

**Comment by citrusjunoss** [comment]
it's not work。
```js
module.export = {
  env: {
    browser: true,
    node: true,
    es6: true,
  },
  parser: 'vue-eslint-parser',
  parserOptions: {
    parser: '@typescript-eslint/parser',
    ecmaVersion: 2020,
    sourceType: 'module',
    project: './tsconfig.*?.json',
    createDefaultProgram: false,
    extraFileExtensions: ['.vue'],
  },
  extends: [
    'eslint:recommended',
    'plugin:vue/vue3-recommended',
    'plugin:@typescript-eslint/recommended',
    'plugin:prettier/recommended',
    "plugin:jsdoc/recommended"
  ],
  rules: {
    'no-unused-vars': 'off',
    'no-case-declarations': 'off',
    'no-use-before-define': 'off',
    'space-before-function-paren': 'off',
    'import/first': 'error',
    'import/newline-after-import': 'error',
    'import/no-duplicates': 'error',
    '@typescript-eslint/no-unused-vars': [
      'error',
      {
        argsIgnorePattern: '^_',
        varsIgnorePattern: '^_',
      },
    ],
    '@typescript-eslint/ban-ts-ignore': 'off',
    '@typescript-eslint/ban-ts-comment': 'off',
    '@typescript-eslint/ban-types': 'off',
    '@typescript-eslint/explicit-function-return-type': 'off',
    '@typescript-eslint/no-explicit-any': 'off',
    '@typescript-eslint/no-var-requires': 'off',
    '@typescript-eslint/no-empty-function': 'off',
    '@typescript-eslint/no-use-before-define': 'off',
    '@typescript-eslint/no-non-null-assertion': 'off',
    '@typescript-eslint/explicit-module-boundary-types': 'off',
    'vue/script-setup-uses-vars': 'error',
    'vue/no-reserved-component-names': 'off',
    'vue/custom-event-name-casing': 'off',
    'vue/attributes-order': 'off',
    'vue/one-component-per-file': 'off',
    'vue/html-closing-bracket-newline': 'off',
    'vue/max-attributes-per-line': 'off',
    'vue/multiline-html-element-content-newline': 'off',
    'vue/singleline-html-element-content-newline': 'off',
    'vue/attribute-hyphenation': 'off',
    'vue/require-default-prop': 'off',
    'vue/require-explicit-emits': 'off',
    'vue/html-self-closing': [
      'error',
      {
        html: {
          void: 'always',
          normal: 'never',
          component: 'always',
        },
        svg: 'always',
        math: 'always',
      },
    ],
    'vue/multi-word-component-names': 'off',
    "max-lines": [
      "error",
      {
        "max": 500, 
        "skipBlankLines": true,
        "skipComments": true
        }
    ],
    "max-depth": ["error", 4],
    "max-lines-per-function": [
      "error", 
      {
        "max": 20, 
        "skipBlankLines": true, 
        "skipComments": true
      }
    ],
    "max-nested-callbacks": ["error", 5]
  },
  globals: { defineOptions: 'readonly' },
};
```

what should I do?
